src/modules/contact/components/form/index.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import styles from "./style.module.css";
+
+const initialState = {
+  firstName: "",
+  lastName: "",
+  phone: "",
+  email: "",
+  subject: "",
+  message: ""
+}
+
+class Form extends Component {
+  constructor() {
+    super();
+    this.state = initialState
+  }
+
+  onFormSubmit = (event) => {
+    event.preventDefault();
+    fetch("https://Som3a.com/", {
+      method: "post",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        firstName: this.state.firstName,
+        lastName: this.state.lastName,
+        phone: this.state.phone,
+        email: this.state.email,
+        subject: this.state.subject,
+        message: this.state.message,
+      }),
+    })
+    .then((response) => response.json())
+    .then((data) => console.log(data))
+    .catch((err) => console.log(err));
+  };
+
+  inputChangeHandler = (event) => {
```

Comment:
Use `name` instead of `id`

Reply:
> Use `name` instead of `id`

what is the difference ?

Reply:
`name` is accessible by `screen readers` and it's a standard to add `name` to the input.

Reply:
Ok, it's not required, u can make it if u want